src/control-plane/tenant-management/tenant-management-funcs.ts

```diff
@@ -45,7 +45,9 @@ export class Services extends Construct {
         {
           id: 'AwsSolutions-IAM5',
           reason: 'Index name(s) not known beforehand.',
-          appliesTo: [`Resource::<ControlPlanetablesstackTenantDetails78527218.Arn>/index/*`],
+          appliesTo: [
+            `Resource::<ControlPlanetenantManagementServicvestenantManagementTableTenantDetails974E95B8.Arn>/index/*`,
```

Comment:
FYI this can be rewritten so it's not something we have to constantly change.

This is something I did elsewhere in this codebase:

```
        {
          id: 'AwsSolutions-IAM5',
          reason: 'Index name(s) not known beforehand.',
          appliesTo: [
            `Resource::<${cdk.Stack.of(this).getLogicalId(meteringRecordsTable.node.defaultChild as dynamodb.CfnTable)}.Arn>/index/*`,
          ],
        },
```

Reply:
Done.